docs/index.txt

```diff
@@ -9,62 +9,172 @@ Laravel MongoDB
 .. meta::
    :keywords: php framework, odm
 
-This package adds functionalities to the Eloquent model and Query builder for
-MongoDB, using the original Laravel API.
-*This library extends the original Laravel classes, so it uses exactly the
-same methods.*
+.. toctree::
+   :titlesonly:
+   :maxdepth: 1
+
+   /install
+   /eloquent-models
+   /query-builder
+   /user-authentication
+   /queues
+   /transactions
+   /upgrade
+
+Introduction
+------------
+
+Welcome to the documentation site for the official {+odm-long+}.
+This package extends methods in the PHP Laravel API to enable you to configure
+your Laravel app to work with MongoDB as a datastore. {+odm-short+} allows you
+to use Laravel's Eloquent syntax and Query Builder to work with your MongoDB
+data.
+
+.. note::
+
+   This documentation describes the ``mongodb/laravel-mongodb`` package,
+   formerly named ``jenssegers/mongodb``. This package is now owned and
+   maintained by MongoDB, Inc. and is compatible with Laravel 10.x and
+   later.
 
-This package was renamed to ``mongodb/laravel-mongodb`` because of a transfer
-of ownership to MongoDB, Inc. It is compatible with Laravel 10.x. For older
-versions of Laravel, please see the `old versions <https://github.com/mongodb/laravel-mongodb/tree/3.9#laravel-version-compatibility>`__.
+   To find versions of the package compatible with older versions of Laravel,
+   see the `Laravel Version Compatibility <https://github.com/mongodb/laravel-mongodb/tree/3.9#laravel-version-compatibility>`__
+   table.
+
+Getting Started
+---------------
+
+Learn how to install and configure your app to MongoDB by using the
+{+odm-short+} in the :ref:`laravel-install` section.
+
+..
+
+   Quick Start
+   -----------
+
+   TODO: this section should replace the Getting Started section once the
+   content is authored.
```

Comment:
Do you want to fill this TODO before merging?

Reply:
I don't have any content to place here yet. While the docs team usually uses RST comments for placeholders, I can remove the comments and store them somewhere else for future use.